### DISCUSSION TITLE
Disabling Certificates Manager in Privacy Settings via GPO

### DISCUSSION THREAD

**Discussion by Bard458** [mozilla/policy-templates#1070] [Q&A]
Hello,

My objective is to disable the option "View Certificates" for all users in our domain except administrators. 

I just enabled the  GPO setting "Preferences" in "User Configuration>Administrative Templates>Mozilla>Firefox" and provided the JSON code that will modify the preference "security.disable_button.openCertManager" 

```
{
  "security.disable_button.openCertManager": {
    "Value": true,
    "Status": "locked"
  },
}
```

But that doesn't work, am i doing something wrong?

**Comment by mkaply** [comment]
You have an extra comma

  },
}

Can you try removing that?

You can go to about:policies to see if there are errors.

**Reply by Bard458** [reply]
sorry but the comma was not in my configuration, also I didn't find any error in about:policies and I noticed that a new entry was added in active policies.
Preferences -> {"security.disable_button.openCertManager": {"Value": true}}

**Comment by mkaply** [comment]
The locked part isn't in about:policies and that's the part that's needed.

Is the  "Status": "locked" somehow missing in your GPO? Can you send me a screenshot in the GPO editor?

**Reply by Bard458** [reply]
![hjbgsizued](https://github.com/mozilla/policy-templates/assets/159452014/573f2f07-f268-4c0d-a94c-73eda5b94f4c)

**Comment by mkaply** [comment]
Misspelled Status :) - Statues should be Status.
